src/semantics/mod.rs

```diff
@@ -576,6 +576,32 @@ where
             ))
         },
 
+        // I-LET
```

Comment:
This refers to an inference rule in `theory.md` that we'll have to update. It's a bit icky to work with (a bunch of mathjax) so I can update it in a separate PR if you like. I have some ideas for overhauling the specification website - see #109 for that!